node-packages/commons/src/api.ts

```diff
@@ -967,6 +967,31 @@ export async function getEnvironmentByOpenshiftProjectName(
   return result;
 }
 
+export async function getBackupEnvironment(
+  openshiftProjectName: string,
+  baas_bucket_name: string
+): Promise<any[]> {
+  const result = await graphqlapi.query(`
+    {
+      backupEnvironment(openshiftProjectName: "${openshiftProjectName}", baas_bucket_name: "${baas_bucket_name}") {
```

Comment:
String interpolation here is pretty dirty (ya, I know you just copy/pasted, sorry). The library supports passing in parameters as a part of the query itself, so that the api can do validation on the server side. It would also prevent some string double escaping issues (acts more like prepared statements). A better example to copy would be [`allProjectsInGroup`](https://github.com/uselagoon/lagoon/blob/main/node-packages/commons/src/api.ts#L638-L653). You would create an outer query that has typed parameters, then pass the paramters as the second argument to `graphqlapi.query`.